source/admin_manual/list_of_events.rst

```diff
@@ -2127,6 +2127,70 @@ Events emitted by dictionary library and dictionary server.
    Dictionary server finishes transaction.
 
 
+Login
+=====
+
+.. dovecot_core:event:: login_aborted
+   :added: v3.0
+
+   :field reason: Short reason, see the short to long reason mapping in the table below.
+   :field auth_successes: Number of successful authentications, which eventually failed due to other reasons.
+   :field auth_attempts: Total number of authentication attempts, both successful and failed.
+   :field auth_usecs: How long ago the first authentication attempt was started.
+   :field connected_usecs: How long ago the client connection was created.
+
+.. list-table::
+   :widths: 25 75
+   :header-rows: 1
+
+   * - Reason
+     - Description
+   * - ``auth_process_not_ready``
+     - Client disconnected before auth process was ready. This may indicate a hanging auth process if connected_usecs is large.
+   * - ``client_ssl_not_started``
+     - Client didn't even start SSL with :dovecot_core:ref:`auth_ssl_require_client_cert` set to ``yes``
+   * - ``client_ssl_cert_untrusted``
+     - Client sent an SSL certificate that is untrusted with :dovecot_core:ref:`auth_ssl_require_client_cert` set to ``yes``
+   * - ``client_ssl_cert_missing``
+     - Client didn't send SSL certificate, but :dovecot_core:ref:`auth_ssl_require_client_cert` is set to ``yes``
+   * - ``no_auth_attempts``
+     - Client didn't send any authentication attempts.
+   * - ``auth_process_comm_fail``
+     - Internal error communicating with the auth process.
+   * - ``auth_waiting_client``
+     - Client started SASL authentication, but disconnected instead of sending the next SASL continuation reply.
+   * - ``process_full``
+     - :ref:`service_configuration-client_limit` and :ref:`service_configuration-process_limit` was hit and this login session was killed.
+   * - ``shutting_down``
+     - The process is shutting down so the login is aborted.
+   * - ``auth_aborted_by_client``
+     - Client started SASL authentication, but returned "*" reply to abort it.
+   * - ``auth_nologin_referral``
+     - Authentication returned auth referral to redirect the client to another server. This is normally configured to be sent only when the client is a Dovecot proxy, which handles the redirection.
+   * - ``proxy_dest_auth_failed``
+     - Local authentication succeeded, but proxying failed to authenticate to the destination hop.
+   * - ``authorization_failed``
+     - Authorization failure
```

Comment:
Maybe: Master user authentication succeeded, but authorization to access the requested login user wasn't allowed.